### DISCUSSION TITLE
Classic Sorc Shield/Tier/Equip improvements

### DISCUSSION THREAD

**Discussion by tyson-dev** [isid0re/SoloLeveling#357] [Ideas]
Is there any way in CLASSIC! to make a 3soc shield with 3Pdiamonds at any point and make that TopTier in classic?

I saw my sorc throwing away a good rare shield with nice @res for that sigon crap shield.
This kept her in NM very long cause of the res-restrictions and also Sigon-Shield makes her pretty weak overall she doesn't improve now @ 69 cause dieing makes her lose more exp than she gains.

I would love to help out with improvements on that tier system or the builds but i don't understand it yet :D

cheers

**Comment by isid0re** [comment]
Its sounds like it's a matter of weights used in dynamictiers.js calculator. As resists improve the value goes down but plus skills done shift in weight. So you might need to adjust them.

Making a 3 pdiamond shield wouldn't be hard. As the code is essentially already there.
